### PR TITLE
Modify PVC Resize UI

### PR DESCRIPTION
**Pull request by am-agrawa**

### CODE REVIEW

File: ocs_ci/ocs/ui/pvc_ui.py [hunk 1]
```diff
@@ -21,14 +23,14 @@ def __init__(self, driver):
         self.pvc_loc = locators[ocp_version]["pvc"]
 
     def create_pvc_ui(
-        self, project_name, sc_type, pvc_name, access_mode, pvc_size, vol_mode
+        self, project_name, sc_name, pvc_name, access_mode, pvc_size, vol_mode
```
Comment (jilju) [inline ocs_ci/ocs/ui/pvc_ui.py:26]: Please make sure no other test is using ```sc_type``` when calling this function.

File: ocs_ci/ocs/ui/pvc_ui.py [hunk 2]
```diff
@@ -21,14 +23,14 @@ def __init__(self, driver):
...
     ):
         """
         Create PVC via UI.
 
         Args:
             project_name (str): name of test project
-            sc_type (str): storage class type
+            sc_name (str): storage class type
```
Comment (jilju) [inline ocs_ci/ocs/ui/pvc_ui.py:33]:
```suggestion
            sc_name (str): storage class name
```

File: ocs_ci/ocs/ui/pvc_ui.py [hunk 3]
```diff
@@ -75,9 +73,10 @@ def create_pvc_ui(
 
         logger.info("Create PVC")
         self.do_click(self.pvc_loc["pvc_create"])
+        time.sleep(1)
 
     def verify_pvc_ui(
-        self, pvc_size, access_mode, vol_mode, sc_type, pvc_name, project_name
+        self, pvc_size, access_mode, vol_mode, sc_name, pvc_name, project_name
```
Comment (jilju) [inline ocs_ci/ocs/ui/pvc_ui.py:79]: Please make sure no other test is using sc_type when calling this function.

File: ocs_ci/ocs/ui/pvc_ui.py [hunk 4]
```diff
@@ -86,7 +85,7 @@ def verify_pvc_ui(
             pvc_size (str): the size of pvc (GB)
             access_mode (str): access mode
             vol_mode (str): volume mode type
-            sc_type (str): storage class type
+            sc_name (str): storage class type
```
Comment (jilju) [inline ocs_ci/ocs/ui/pvc_ui.py:88]:
```suggestion
            sc_name (str): storage class name
```

File: ocs_ci/ocs/ui/pvc_ui.py [hunk 5]
```diff
@@ -86,7 +85,7 @@ def verify_pvc_ui(
...
             pvc_name (str): the name of pvc
             project_name (str): name of test project
```
Comment (jilju) [inline ocs_ci/ocs/ui/pvc_ui.py:91]: Only one empty line is needed.

**Review by OdedViner — APPROVED** [review]

**Review by ebenahar — APPROVED** [review]

**Pull request merged**